react/week4/src/components/UserList.js

```diff
@@ -0,0 +1,32 @@
+import React, { useContext } from "react";
+import { userListContext } from "../App";
+import EmptyResult from "./EmptyResult";
+
+const UserList = () => {
+  const { user } = useContext(userListContext);
+  return (
+    <ul>
+      {user.total_count === undefined ? (
+        <EmptyResult />
+      ) : (
+        user.items.map(list => {

```

Comment:
Please apply the same updates as suggested for the map in the `MyRepos` component.

Reply:
corrected